New package: StrassOPen v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: StrassOPen
- Repository: https://github.com/bakerte/StrassOPen.jl
- Created by: @bakerte
- Version: v0.1.0
- Commit: 79c4245f416137c20e0ee3e319f146da472f035b
- Git reference: v0.4.0
- Description: An open source implementation of the Strassen algorithm
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Basic implementation of the Strassen algorithm (non-recursive)
`````
<!-- END RELEASE NOTES -->

ChrisRackauckas commented:
The package doesn't have any code? https://github.com/bakerte/StrassOPen.jl/blob/main/src/StrassOPen.jl it's importing files that don't exist and fails on `using`?

bakerte commented:
Hi Chris,

Sorry about that. I didn’t even think about it. We are adding a dependency onto TensorPACK.jl (itself a dependency of DMRjulia for reasons of better version control), and I figured to just register the package so I could ensure I got the UUIDs and everything set up correctly for TensorPACK (I’ve never added a dependency after the fact like this before). Then I had it in mind to add the real source code afterwards. Maybe I should have made it simpler.

If you would prefer, I could push the rest of the code for the initial commit. I just thought having nothing there would make it easier to ensure that I got everything tied together correctly. If not, we are likely to push everything next week and submit a paper sometime this month. This package should have much wider use than the others that have a dependency on it, hence the split of the code.

PS You gave an introduction to Julia at UCI that I attended approximately many years ago. Great to reconnect with you.

> On Jan 1, 2025, at 04:03, Christopher Rackauckas ***@***.***> wrote:
> 
> 
> The package doesn't have any code? https://github.com/bakerte/StrassOPen.jl/blob/main/src/StrassOPen.jl it's importing files that don't exist and fails on using?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/JuliaRegistries/General/pull/122240#issuecomment-2566979201>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AB3JZYXLCNHTD3LOMLUMIIT2IPKP5AVCNFSM6AAAAABUOPC2QSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRWHE3TSMRQGE>.
> You are receiving this because you were mentioned.
>

JuliaHubRegistrator committed:
New package: StrassOPen v0.1.0

UUID: 1971c852-c825-11ef-1d57-2185ebdca1d8
Repo: https://github.com/bakerte/StrassOPen.jl.git
Tree: c75355db397b833eadb97b6a4f64668c6bafd4f3

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
